app/src/main/java/com/team/uptech/pomodoro/presentation/presenter/SettingsPresenter.kt

```diff
@@ -0,0 +1,15 @@
+package com.team.uptech.pomodoro.presentation.presenter
+
+import com.team.uptech.pomodoro.data.model.Pomodoro
+import com.team.uptech.pomodoro.presentation.ui.view.SettingsView
+
+/**
+ * Created on 18.05.17.
+ */
+interface SettingsPresenter : BasePresenter<SettingsView> {
+    fun getPomodoroTime(pomodoro: Pomodoro)
+    fun getIsInfinite()
```

Comment:
who calls these get() methods? view should not get info, it should say "hey, I'm here!" and the presenter decides what to do